packages/module/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@patternfly/react-console",
+  "version": "4.93.25",
+  "description": "This package provides VncConsole, SerialConsole and DesktopViewer React components to be used alongside patternfly-react to access virtual machine or server consoles.",
+  "main": "dist/esm/index.js",
+  "module": "dist/esm/index.js",
+  "scripts": {
+    "build": "tsc --build --verbose ./tsconfig.json",
+    "clean": "rimraf dist",
+    "docs:develop": "pf-docs-framework start",
+    "docs:build": "pf-docs-framework build all --output public",
+    "docs:serve": "pf-docs-framework serve public --port 5000",
+    "docs:screenshots": "pf-docs-framework screenshots --urlPrefix http://localhost:5000",
+    "test:a11y": "patternfly-a11y --config patternfly-a11y.config",
+    "serve:a11y": "yarn serve coverage"
+  },
+  "repository": "git+https://github.com/patternfly/patternfly-react/.git",
+  "keywords": [
+    "react",
+    "patternfly",
+    "console"
+  ],
+  "author": "Red Hat",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/patternfly/react-console//issues"
+  },
+  "homepage": "https://github.com/patternfly/react-console/#readme",
+  "release": {
+    "branches": "main"
+  },
```

Comment:
You'll probably want to delete this key and add a [release config file](https://github.com/patternfly/react-catalog-view/blob/main/packages/module/release.config.js). Additionally I would probably enable `dryRun` in that config file until we know that the release process works like we want it to.